src/lwc/geFormRenderer/geFormRenderer.js

```diff
@@ -65,16 +75,35 @@ export default class GeFormRenderer extends NavigationMixin(LightningElement) {
         GeFormService.getFormTemplate().then(response => {
             // read the template header info
             if(response !== null && typeof response !== 'undefined') {
-                this.formTemplate  = response.formTemplate;
-                this.fieldMappings = response.fieldMappingSetWrapper.fieldMappingByDevName;
-
-                // get the target field names to be used by getRecord
-                this.fieldNames = getRecordFieldNames(this.formTemplate, this.fieldMappings, donorApiName);
-            
-                if(isEmpty(this.donorRecordId)) {
-                    // if we don't have a donor record, it's ok to initialize the form now
-                    // otherwise the form will be initialized after wiredGetRecordMethod completes
-                    this.initializeForm(this.formTemplate);
+                let template = response.formTemplate;
```

Comment:
Do we need another local variable template, we already had the 'this.formTemplate  = response.formTemplate;' line above.

Reply:
fixed